src/inc/request-handler.ts

```diff
@@ -4,6 +4,12 @@ import ExpiredAuthSessionError from './expired-auth-session-error'
 export default class RequestHandler {
   constructor (public $auth: Auth) {}
 
+  _needToken (config: HTTPRequest) {
+    const options = this.$auth.strategy.options
+    return options.token.global || Object.values(options.endpoints)
+      .some(endpoint => typeof endpoint === 'object' ? endpoint.url === config.url : endpoint === config.url)
```

Comment:
To fix the error `Property 'url' does not exist on type 'object'`
```suggestion
      .some((endpoint: HTTPRequest | string) => typeof endpoint === 'object' ? endpoint.url === config.url : endpoint === config.url)
```